articles/multifactor-authentication/google-auth/admin-guide.md

```diff
@@ -4,6 +4,10 @@ description:  Using Google Authenticator with Auth0 for administrators
 
 # Google Authenticator for Administrators
 
+::: warn
```

Comment:
Should be
```
::: warning
text here...
::::
```

Reply:
Thanks! Can't figure why I thought it was `warn`!